doppler/models.go

```diff
@@ -207,3 +207,23 @@ func parseServiceTokenResourceId(id string) (project string, config string, slug
 	}
 	return tokens[0], tokens[1], tokens[2], nil
 }
+
+type ServiceAccountWorkplaceRole struct {
+	Name         string   `json:"name"`
+	Permissions  []string `json:"permissions"`
+	Identifier   string   `json:"identifier,omitempty"`
+	IsCustomRole bool     `json:"is_custom_role"`
+	IsInlineRole bool     `json:"is_inline_role"`
+	CreatedAt    string   `json:"created_at"`
+}
+
+type ServiceAccount struct {
+	Slug          string                      `json:"slug"`
+	Name          string                      `json:"name"`
+	CreatedAt     string                      `json:"created_at"`
+	WorkplaceRole ServiceAccountWorkplaceRole `json:"workplace_role"`
+}
```

Comment:
On the backend we have a single WorkplaceRoleDTO for service accounts, users, invites, etc. It's the same DTO we use for our actual workplace role endpoint (which we'll add a terraform resource for in the future). You can thus have a single `WorkplaceRole` type if you'd prefer, but totally up to you.

Reply:
Precisely - just that we'll be able to use this more broadly when we inevitable add additional resources that contain a workplace role.

Reply:
Sounds good! I'll make the change